File: java/0217-contains-duplicate/Main.java
public class Main {
    public static void main(String[] args) {
        Solution solution = new Solution();

        System.out.println("true: " + solution.containsDuplicate(new int[] { 1, 2, 3, 1 }));
        System.out.println("false: " + solution.containsDuplicate(new int[] { 1, 2, 3, 4 }));
        System.out.println("true: " + solution.containsDuplicate(new int[] { 1, 1, 1, 3, 3, 4, 3, 2, 4, 2 }));
    }
}

File: c++/0217-contains-duplicate/main.cpp
#include <cstdlib>
#include <iostream>
#include <string>
#include <vector>

#include "solution.cpp"

using namespace std;

int main() {
    Solution solution;

    vector<int> nums1 = {1, 2, 3, 1};
    string solution1 = solution.containsDuplicate(nums1) ? "true" : "false";
    cout << "true: " << solution1 << endl;

    vector<int> nums2 = {1, 2, 3, 4};
    string solution2 = solution.containsDuplicate(nums2) ? "true" : "false";
    cout << "false: " << solution2 << endl;

    vector<int> nums3 = {1, 1, 1, 3, 3, 4, 3, 2, 4, 2};
    string solution3 = solution.containsDuplicate(nums3) ? "true" : "false";
    cout << "true: " << solution3 << endl;

    return EXIT_SUCCESS;
}
